ocs_ci/framework/tests/order/a/test_a.py

```diff
@@ -0,0 +1,19 @@
+from os import environ
+from ocs_ci.framework.pytest_customization.marks import (
+    post_ocs_upgrade, post_upgrade, ocs_upgrade
+)
+
+
+@post_upgrade
+def test_1():
+    assert environ['RH'] != "Red Hat"
```

Comment:
I'm not sure I understand how does these unit tests work. Could we drop a note in a comment or docstring there?

Reply:
I will add the docstring explaining the test

Reply:
fixed